diff --git a/Assets/Project/Animations/Bat/Hit.anim b/Assets/Project/Animations/Bat/Hit.anim
index 6ab56b9..9719af4 100644
--- a/Assets/Project/Animations/Bat/Hit.anim
+++ b/Assets/Project/Animations/Bat/Hit.anim
@@ -19,11 +19,19 @@ AnimationClip:
   m_PPtrCurves:
   - curve:
     - time: 0
+<<<<<<< HEAD
       value: {fileID: 21300034, guid: 32210791b50e292438a60b04f970b3fe, type: 3}
     - time: 0.16666667
       value: {fileID: 21300036, guid: 32210791b50e292438a60b04f970b3fe, type: 3}
     - time: 0.33333334
       value: {fileID: 21300036, guid: 32210791b50e292438a60b04f970b3fe, type: 3}
+=======
+      value: {fileID: 21300034, guid: 568afcd578cf0804699e5a55a488e05b, type: 3}
+    - time: 0.16666667
+      value: {fileID: 21300036, guid: 568afcd578cf0804699e5a55a488e05b, type: 3}
+    - time: 0.33333334
+      value: {fileID: 21300036, guid: 568afcd578cf0804699e5a55a488e05b, type: 3}
+>>>>>>> 0add8808888a7ca2fba02d9b1f3db450008863fb
     attribute: m_Sprite
     path: 
     classID: 212
@@ -43,9 +51,15 @@ AnimationClip:
       customType: 23
       isPPtrCurve: 1
     pptrCurveMapping:
+<<<<<<< HEAD
     - {fileID: 21300034, guid: 32210791b50e292438a60b04f970b3fe, type: 3}
     - {fileID: 21300036, guid: 32210791b50e292438a60b04f970b3fe, type: 3}
     - {fileID: 21300036, guid: 32210791b50e292438a60b04f970b3fe, type: 3}
+=======
+    - {fileID: 21300034, guid: 568afcd578cf0804699e5a55a488e05b, type: 3}
+    - {fileID: 21300036, guid: 568afcd578cf0804699e5a55a488e05b, type: 3}
+    - {fileID: 21300036, guid: 568afcd578cf0804699e5a55a488e05b, type: 3}
+>>>>>>> 0add8808888a7ca2fba02d9b1f3db450008863fb
   m_AnimationClipSettings:
     serializedVersion: 2
     m_AdditiveReferencePoseClip: {fileID: 0}
@@ -56,7 +70,11 @@ AnimationClip:
     m_Level: 0
     m_CycleOffset: 0
     m_HasAdditiveReferencePose: 0
+<<<<<<< HEAD
     m_LoopTime: 1
+=======
+    m_LoopTime: 0
+>>>>>>> 0add8808888a7ca2fba02d9b1f3db450008863fb
     m_LoopBlend: 0
     m_LoopBlendOrientation: 0
     m_LoopBlendPositionY: 0
diff --git a/Assets/Project/Animations/Bat/Hit.anim.meta b/Assets/Project/Animations/Bat/Hit.anim.meta
index 0c56a2e..fb8d51d 100644
--- a/Assets/Project/Animations/Bat/Hit.anim.meta
+++ b/Assets/Project/Animations/Bat/Hit.anim.meta
@@ -1,7 +1,14 @@
 fileFormatVersion: 2
+<<<<<<< HEAD
 guid: d0afef68850632d4abd4945e3a176c94
 NativeFormatImporter:
   externalObjects: {}
+=======
+guid: fbfd811d780fc3044ad9968e16a0d4e9
+timeCreated: 1497442615
+licenseType: Free
+NativeFormatImporter:
+>>>>>>> 0add8808888a7ca2fba02d9b1f3db450008863fb
   mainObjectFileID: 7400000
   userData: 
   assetBundleName: 
diff --git a/Assets/Project/Animations/Bat/Idle.anim b/Assets/Project/Animations/Bat/Idle.anim
index 0b4b331..2d65cd3 100644
--- a/Assets/Project/Animations/Bat/Idle.anim
+++ b/Assets/Project/Animations/Bat/Idle.anim
@@ -19,6 +19,7 @@ AnimationClip:
   m_PPtrCurves:
   - curve:
     - time: 0
+<<<<<<< HEAD
       value: {fileID: 21300000, guid: 32210791b50e292438a60b04f970b3fe, type: 3}
     - time: 0.16666667
       value: {fileID: 21300002, guid: 32210791b50e292438a60b04f970b3fe, type: 3}
@@ -32,6 +33,19 @@ AnimationClip:
       value: {fileID: 21300010, guid: 32210791b50e292438a60b04f970b3fe, type: 3}
     - time: 1
       value: {fileID: 21300010, guid: 32210791b50e292438a60b04f970b3fe, type: 3}
+=======
+      value: {fileID: 21300000, guid: 568afcd578cf0804699e5a55a488e05b, type: 3}
+    - time: 0.16666667
+      value: {fileID: 21300002, guid: 568afcd578cf0804699e5a55a488e05b, type: 3}
+    - time: 0.33333334
+      value: {fileID: 21300004, guid: 568afcd578cf0804699e5a55a488e05b, type: 3}
+    - time: 0.5
+      value: {fileID: 21300006, guid: 568afcd578cf0804699e5a55a488e05b, type: 3}
+    - time: 0.6666667
+      value: {fileID: 21300008, guid: 568afcd578cf0804699e5a55a488e05b, type: 3}
+    - time: 0.8333333
+      value: {fileID: 21300010, guid: 568afcd578cf0804699e5a55a488e05b, type: 3}
+>>>>>>> 0add8808888a7ca2fba02d9b1f3db450008863fb
     attribute: m_Sprite
     path: 
     classID: 212
@@ -51,6 +65,7 @@ AnimationClip:
       customType: 23
       isPPtrCurve: 1
     pptrCurveMapping:
+<<<<<<< HEAD
     - {fileID: 21300000, guid: 32210791b50e292438a60b04f970b3fe, type: 3}
     - {fileID: 21300002, guid: 32210791b50e292438a60b04f970b3fe, type: 3}
     - {fileID: 21300004, guid: 32210791b50e292438a60b04f970b3fe, type: 3}
@@ -58,12 +73,24 @@ AnimationClip:
     - {fileID: 21300008, guid: 32210791b50e292438a60b04f970b3fe, type: 3}
     - {fileID: 21300010, guid: 32210791b50e292438a60b04f970b3fe, type: 3}
     - {fileID: 21300010, guid: 32210791b50e292438a60b04f970b3fe, type: 3}
+=======
+    - {fileID: 21300000, guid: 568afcd578cf0804699e5a55a488e05b, type: 3}
+    - {fileID: 21300002, guid: 568afcd578cf0804699e5a55a488e05b, type: 3}
+    - {fileID: 21300004, guid: 568afcd578cf0804699e5a55a488e05b, type: 3}
+    - {fileID: 21300006, guid: 568afcd578cf0804699e5a55a488e05b, type: 3}
+    - {fileID: 21300008, guid: 568afcd578cf0804699e5a55a488e05b, type: 3}
+    - {fileID: 21300010, guid: 568afcd578cf0804699e5a55a488e05b, type: 3}
+>>>>>>> 0add8808888a7ca2fba02d9b1f3db450008863fb
   m_AnimationClipSettings:
     serializedVersion: 2
     m_AdditiveReferencePoseClip: {fileID: 0}
     m_AdditiveReferencePoseTime: 0
     m_StartTime: 0
+<<<<<<< HEAD
     m_StopTime: 1.0166667
+=======
+    m_StopTime: 0.84999996
+>>>>>>> 0add8808888a7ca2fba02d9b1f3db450008863fb
     m_OrientationOffsetY: 0
     m_Level: 0
     m_CycleOffset: 0
diff --git a/Assets/Project/Animations/Bat/Idle.anim.meta b/Assets/Project/Animations/Bat/Idle.anim.meta
index a58c982..2b9b98c 100644
--- a/Assets/Project/Animations/Bat/Idle.anim.meta
+++ b/Assets/Project/Animations/Bat/Idle.anim.meta
@@ -1,7 +1,14 @@
 fileFormatVersion: 2
+<<<<<<< HEAD
 guid: d4eb283232747a94ca11cce48dee0999
 NativeFormatImporter:
   externalObjects: {}
+=======
+guid: 06436bfa8eaddd747b5717fa04f61043
+timeCreated: 1497442615
+licenseType: Free
+NativeFormatImporter:
+>>>>>>> 0add8808888a7ca2fba02d9b1f3db450008863fb
   mainObjectFileID: 7400000
   userData: 
   assetBundleName: 
diff --git a/Assets/Project/Animations/Bat/PhysicalAttack.anim b/Assets/Project/Animations/Bat/PhysicalAttack.anim
index f2e968a..e1334ef 100644
--- a/Assets/Project/Animations/Bat/PhysicalAttack.anim
+++ b/Assets/Project/Animations/Bat/PhysicalAttack.anim
@@ -19,6 +19,7 @@ AnimationClip:
   m_PPtrCurves:
   - curve:
     - time: 0
+<<<<<<< HEAD
       value: {fileID: 21300022, guid: 32210791b50e292438a60b04f970b3fe, type: 3}
     - time: 0.16666667
       value: {fileID: 21300024, guid: 32210791b50e292438a60b04f970b3fe, type: 3}
@@ -32,6 +33,17 @@ AnimationClip:
       value: {fileID: 21300032, guid: 32210791b50e292438a60b04f970b3fe, type: 3}
     - time: 1
       value: {fileID: 21300032, guid: 32210791b50e292438a60b04f970b3fe, type: 3}
+=======
+      value: {fileID: 21300012, guid: 568afcd578cf0804699e5a55a488e05b, type: 3}
+    - time: 0.16666667
+      value: {fileID: 21300014, guid: 568afcd578cf0804699e5a55a488e05b, type: 3}
+    - time: 0.33333334
+      value: {fileID: 21300016, guid: 568afcd578cf0804699e5a55a488e05b, type: 3}
+    - time: 0.5
+      value: {fileID: 21300018, guid: 568afcd578cf0804699e5a55a488e05b, type: 3}
+    - time: 0.6666667
+      value: {fileID: 21300020, guid: 568afcd578cf0804699e5a55a488e05b, type: 3}
+>>>>>>> 0add8808888a7ca2fba02d9b1f3db450008863fb
     attribute: m_Sprite
     path: 
     classID: 212
@@ -51,6 +63,7 @@ AnimationClip:
       customType: 23
       isPPtrCurve: 1
     pptrCurveMapping:
+<<<<<<< HEAD
     - {fileID: 21300022, guid: 32210791b50e292438a60b04f970b3fe, type: 3}
     - {fileID: 21300024, guid: 32210791b50e292438a60b04f970b3fe, type: 3}
     - {fileID: 21300026, guid: 32210791b50e292438a60b04f970b3fe, type: 3}
@@ -58,17 +71,32 @@ AnimationClip:
     - {fileID: 21300030, guid: 32210791b50e292438a60b04f970b3fe, type: 3}
     - {fileID: 21300032, guid: 32210791b50e292438a60b04f970b3fe, type: 3}
     - {fileID: 21300032, guid: 32210791b50e292438a60b04f970b3fe, type: 3}
+=======
+    - {fileID: 21300012, guid: 568afcd578cf0804699e5a55a488e05b, type: 3}
+    - {fileID: 21300014, guid: 568afcd578cf0804699e5a55a488e05b, type: 3}
+    - {fileID: 21300016, guid: 568afcd578cf0804699e5a55a488e05b, type: 3}
+    - {fileID: 21300018, guid: 568afcd578cf0804699e5a55a488e05b, type: 3}
+    - {fileID: 21300020, guid: 568afcd578cf0804699e5a55a488e05b, type: 3}
+>>>>>>> 0add8808888a7ca2fba02d9b1f3db450008863fb
   m_AnimationClipSettings:
     serializedVersion: 2
     m_AdditiveReferencePoseClip: {fileID: 0}
     m_AdditiveReferencePoseTime: 0
     m_StartTime: 0
+<<<<<<< HEAD
     m_StopTime: 1.0166667
+=======
+    m_StopTime: 0.68333334
+>>>>>>> 0add8808888a7ca2fba02d9b1f3db450008863fb
     m_OrientationOffsetY: 0
     m_Level: 0
     m_CycleOffset: 0
     m_HasAdditiveReferencePose: 0
+<<<<<<< HEAD
     m_LoopTime: 1
+=======
+    m_LoopTime: 0
+>>>>>>> 0add8808888a7ca2fba02d9b1f3db450008863fb
     m_LoopBlend: 0
     m_LoopBlendOrientation: 0
     m_LoopBlendPositionY: 0
diff --git a/Assets/Project/Animations/Bat/PhysicalAttack.anim.meta b/Assets/Project/Animations/Bat/PhysicalAttack.anim.meta
index 258324d..08e1b80 100644
--- a/Assets/Project/Animations/Bat/PhysicalAttack.anim.meta
+++ b/Assets/Project/Animations/Bat/PhysicalAttack.anim.meta
@@ -1,7 +1,14 @@
 fileFormatVersion: 2
+<<<<<<< HEAD
 guid: 8410c7bf2eff0ee4b9e5222a406d3906
 NativeFormatImporter:
   externalObjects: {}
+=======
+guid: 8e391f4f137a5794f8b4c2b2f2d5fe9b
+timeCreated: 1497442615
+licenseType: Free
+NativeFormatImporter:
+>>>>>>> 0add8808888a7ca2fba02d9b1f3db450008863fb
   mainObjectFileID: 7400000
   userData: 
   assetBundleName: 
diff --git a/Assets/Project/Animations/Snake/MagicalAttack.anim b/Assets/Project/Animations/Snake/MagicalAttack.anim
index 39f86ad..23a93a1 100644
--- a/Assets/Project/Animations/Snake/MagicalAttack.anim
+++ b/Assets/Project/Animations/Snake/MagicalAttack.anim
@@ -16,21 +16,54 @@ AnimationClip:
   m_PositionCurves: []
   m_ScaleCurves: []
   m_FloatCurves: []
-  m_PPtrCurves: []
+  m_PPtrCurves:
+  - curve:
+    - time: 0
+      value: {fileID: 21300022, guid: 568afcd578cf0804699e5a55a488e05b, type: 3}
+    - time: 0.16666667
+      value: {fileID: 21300024, guid: 568afcd578cf0804699e5a55a488e05b, type: 3}
+    - time: 0.33333334
+      value: {fileID: 21300026, guid: 568afcd578cf0804699e5a55a488e05b, type: 3}
+    - time: 0.5
+      value: {fileID: 21300028, guid: 568afcd578cf0804699e5a55a488e05b, type: 3}
+    - time: 0.6666667
+      value: {fileID: 21300030, guid: 568afcd578cf0804699e5a55a488e05b, type: 3}
+    - time: 0.8333333
+      value: {fileID: 21300032, guid: 568afcd578cf0804699e5a55a488e05b, type: 3}
+    - time: 1
+      value: {fileID: 21300032, guid: 568afcd578cf0804699e5a55a488e05b, type: 3}
+    attribute: m_Sprite
+    path: 
+    classID: 212
+    script: {fileID: 0}
   m_SampleRate: 60
   m_WrapMode: 0
   m_Bounds:
     m_Center: {x: 0, y: 0, z: 0}
     m_Extent: {x: 0, y: 0, z: 0}
   m_ClipBindingConstant:
-    genericBindings: []
-    pptrCurveMapping: []
+    genericBindings:
+    - serializedVersion: 2
+      path: 0
+      attribute: 0
+      script: {fileID: 0}
+      typeID: 212
+      customType: 23
+      isPPtrCurve: 1
+    pptrCurveMapping:
+    - {fileID: 21300022, guid: 568afcd578cf0804699e5a55a488e05b, type: 3}
+    - {fileID: 21300024, guid: 568afcd578cf0804699e5a55a488e05b, type: 3}
+    - {fileID: 21300026, guid: 568afcd578cf0804699e5a55a488e05b, type: 3}
+    - {fileID: 21300028, guid: 568afcd578cf0804699e5a55a488e05b, type: 3}
+    - {fileID: 21300030, guid: 568afcd578cf0804699e5a55a488e05b, type: 3}
+    - {fileID: 21300032, guid: 568afcd578cf0804699e5a55a488e05b, type: 3}
+    - {fileID: 21300032, guid: 568afcd578cf0804699e5a55a488e05b, type: 3}
   m_AnimationClipSettings:
     serializedVersion: 2
     m_AdditiveReferencePoseClip: {fileID: 0}
     m_AdditiveReferencePoseTime: 0
     m_StartTime: 0
-    m_StopTime: 1
+    m_StopTime: 1.0166667
     m_OrientationOffsetY: 0
     m_Level: 0
     m_CycleOffset: 0
diff --git a/Assets/Project/Prefabs/EnemyEncounters/SnakeEnemyEncounter.prefab b/Assets/Project/Prefabs/EnemyEncounters/SnakeEnemyEncounter.prefab
index 2ed658b..a7c99b7 100644
--- a/Assets/Project/Prefabs/EnemyEncounters/SnakeEnemyEncounter.prefab
+++ b/Assets/Project/Prefabs/EnemyEncounters/SnakeEnemyEncounter.prefab
@@ -33,6 +33,7 @@ GameObject:
   m_PrefabInternal: {fileID: 100100000}
   serializedVersion: 6
   m_Component:
+<<<<<<< HEAD:Assets/Project/Prefabs/EnemyEncounters/SnakeEnemyEncounter.prefab
   - component: {fileID: 4472234633737834}
   - component: {fileID: 212286205727974072}
   - component: {fileID: 114114402268787076}
@@ -40,6 +41,10 @@ GameObject:
   - component: {fileID: 95886074107247398}
   - component: {fileID: 114278138636820220}
   - component: {fileID: 114440016505162300}
+=======
+  - component: {fileID: 4580493608097234}
+  - component: {fileID: 114416190795202166}
+>>>>>>> 0add8808888a7ca2fba02d9b1f3db450008863fb:Assets/Project/Prefabs/EnemyEncounters/SnakeEncounter.prefab
   m_Layer: 0
   m_Name: SnakeUnit (1)
   m_TagString: EnemyUnit
@@ -184,7 +189,23 @@ MonoBehaviour:
   defense: 5
   speed: 10
   nextActTurn: 0
+<<<<<<< HEAD:Assets/Project/Prefabs/EnemyEncounters/SnakeEnemyEncounter.prefab
 --- !u!114 &114278138636820220
+=======
+--- !u!114 &114416190795202166
+MonoBehaviour:
+  m_ObjectHideFlags: 1
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInternal: {fileID: 100100000}
+  m_GameObject: {fileID: 1413717556820096}
+  m_Enabled: 1
+  m_EditorHideFlags: 0
+  m_Script: {fileID: 11500000, guid: 825515865f962ef428af3836565adf69, type: 3}
+  m_Name: 
+  m_EditorClassIdentifier: 
+  experience: 100
+--- !u!114 &114510171833866140
+>>>>>>> 0add8808888a7ca2fba02d9b1f3db450008863fb:Assets/Project/Prefabs/EnemyEncounters/SnakeEncounter.prefab
 MonoBehaviour:
   m_ObjectHideFlags: 1
   m_CorrespondingSourceObject: {fileID: 0}
diff --git a/Assets/Project/Prefabs/EnemyUnits/BatUnit.prefab b/Assets/Project/Prefabs/EnemyUnits/BatUnit.prefab
index 8f2f96c..0d263e2 100644
--- a/Assets/Project/Prefabs/EnemyUnits/BatUnit.prefab
+++ b/Assets/Project/Prefabs/EnemyUnits/BatUnit.prefab
@@ -9,15 +9,22 @@ Prefab:
     m_Modifications: []
     m_RemovedComponents: []
   m_SourcePrefab: {fileID: 0}
+<<<<<<< HEAD
   m_RootGameObject: {fileID: 1666034286474376}
   m_IsPrefabAsset: 1
 --- !u!1 &1666034286474376
+=======
+  m_RootGameObject: {fileID: 1709345393717716}
+  m_IsPrefabAsset: 1
+--- !u!1 &1709345393717716
+>>>>>>> 0add8808888a7ca2fba02d9b1f3db450008863fb
 GameObject:
   m_ObjectHideFlags: 0
   m_CorrespondingSourceObject: {fileID: 0}
   m_PrefabInternal: {fileID: 100100000}
   serializedVersion: 6
   m_Component:
+<<<<<<< HEAD
   - component: {fileID: 4434809226941404}
   - component: {fileID: 212274055380817034}
   - component: {fileID: 114594426176200700}
@@ -25,6 +32,15 @@ GameObject:
   - component: {fileID: 95980838961599054}
   - component: {fileID: 114578510678012218}
   - component: {fileID: 114691746488550066}
+=======
+  - component: {fileID: 4210981096895380}
+  - component: {fileID: 212709489256783472}
+  - component: {fileID: 114628641955667000}
+  - component: {fileID: 114421328445299954}
+  - component: {fileID: 114494258235493402}
+  - component: {fileID: 95498335459604692}
+  - component: {fileID: 114034877023520200}
+>>>>>>> 0add8808888a7ca2fba02d9b1f3db450008863fb
   m_Layer: 0
   m_Name: BatUnit
   m_TagString: EnemyUnit
@@ -32,29 +48,50 @@ GameObject:
   m_NavMeshLayer: 0
   m_StaticEditorFlags: 0
   m_IsActive: 1
+<<<<<<< HEAD
 --- !u!4 &4434809226941404
+=======
+--- !u!4 &4210981096895380
+>>>>>>> 0add8808888a7ca2fba02d9b1f3db450008863fb
 Transform:
   m_ObjectHideFlags: 1
   m_CorrespondingSourceObject: {fileID: 0}
   m_PrefabInternal: {fileID: 100100000}
+<<<<<<< HEAD
   m_GameObject: {fileID: 1666034286474376}
   m_LocalRotation: {x: -0, y: -0, z: -0, w: 1}
   m_LocalPosition: {x: -3.91, y: 0.88, z: 5.10327}
+=======
+  m_GameObject: {fileID: 1709345393717716}
+  m_LocalRotation: {x: -0, y: -0, z: -0, w: 1}
+  m_LocalPosition: {x: -3.033311, y: 0.7190585, z: -38.02832}
+>>>>>>> 0add8808888a7ca2fba02d9b1f3db450008863fb
   m_LocalScale: {x: 1, y: 1, z: 1}
   m_Children: []
   m_Father: {fileID: 0}
   m_RootOrder: 0
   m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
+<<<<<<< HEAD
 --- !u!95 &95980838961599054
+=======
+--- !u!95 &95498335459604692
+>>>>>>> 0add8808888a7ca2fba02d9b1f3db450008863fb
 Animator:
   serializedVersion: 3
   m_ObjectHideFlags: 1
   m_CorrespondingSourceObject: {fileID: 0}
   m_PrefabInternal: {fileID: 100100000}
+<<<<<<< HEAD
   m_GameObject: {fileID: 1666034286474376}
   m_Enabled: 1
   m_Avatar: {fileID: 0}
   m_Controller: {fileID: 9100000, guid: 8bc04fff41407494a8994a939028ff84, type: 2}
+=======
+  m_GameObject: {fileID: 1709345393717716}
+  m_Enabled: 1
+  m_Avatar: {fileID: 0}
+  m_Controller: {fileID: 9100000, guid: c679b547c08143144b62e2cb5546e75c, type: 2}
+>>>>>>> 0add8808888a7ca2fba02d9b1f3db450008863fb
   m_CullingMode: 0
   m_UpdateMode: 0
   m_ApplyRootMotion: 0
@@ -63,11 +100,16 @@ Animator:
   m_HasTransformHierarchy: 1
   m_AllowConstantClipSamplingOptimization: 1
   m_KeepAnimatorControllerStateOnDisable: 0
+<<<<<<< HEAD
 --- !u!114 &114578510678012218
+=======
+--- !u!114 &114034877023520200
+>>>>>>> 0add8808888a7ca2fba02d9b1f3db450008863fb
 MonoBehaviour:
   m_ObjectHideFlags: 1
   m_CorrespondingSourceObject: {fileID: 0}
   m_PrefabInternal: {fileID: 100100000}
+<<<<<<< HEAD
   m_GameObject: {fileID: 1666034286474376}
   m_Enabled: 1
   m_EditorHideFlags: 0
@@ -76,10 +118,20 @@ MonoBehaviour:
   m_EditorClassIdentifier: 
   menuItem: {fileID: 0}
 --- !u!114 &114594426176200700
+=======
+  m_GameObject: {fileID: 1709345393717716}
+  m_Enabled: 1
+  m_EditorHideFlags: 0
+  m_Script: {fileID: 11500000, guid: 43c11fab837707447af8d339d3f6a4a2, type: 3}
+  m_Name: 
+  m_EditorClassIdentifier: 
+--- !u!114 &114421328445299954
+>>>>>>> 0add8808888a7ca2fba02d9b1f3db450008863fb
 MonoBehaviour:
   m_ObjectHideFlags: 1
   m_CorrespondingSourceObject: {fileID: 0}
   m_PrefabInternal: {fileID: 100100000}
+<<<<<<< HEAD
   m_GameObject: {fileID: 1666034286474376}
   m_Enabled: 1
   m_EditorHideFlags: 0
@@ -97,10 +149,21 @@ MonoBehaviour:
   speed: 20
   nextActTurn: 0
 --- !u!114 &114691746488550066
+=======
+  m_GameObject: {fileID: 1709345393717716}
+  m_Enabled: 1
+  m_EditorHideFlags: 0
+  m_Script: {fileID: 11500000, guid: 74602528d72b36e46a97629b82b4fb3f, type: 3}
+  m_Name: 
+  m_EditorClassIdentifier: 
+  menuItem: {fileID: 0}
+--- !u!114 &114494258235493402
+>>>>>>> 0add8808888a7ca2fba02d9b1f3db450008863fb
 MonoBehaviour:
   m_ObjectHideFlags: 1
   m_CorrespondingSourceObject: {fileID: 0}
   m_PrefabInternal: {fileID: 100100000}
+<<<<<<< HEAD
   m_GameObject: {fileID: 1666034286474376}
   m_Enabled: 1
   m_EditorHideFlags: 0
@@ -108,17 +171,32 @@ MonoBehaviour:
   m_Name: 
   m_EditorClassIdentifier: 
   targetEnemyUnitPrefab: {fileID: 1800675918331624, guid: 7d7823586617e0d409a8ac31ba23a627,
+=======
+  m_GameObject: {fileID: 1709345393717716}
+  m_Enabled: 1
+  m_EditorHideFlags: 0
+  m_Script: {fileID: 11500000, guid: d1e36b0d74cc5de4ea03cd7544bf2378, type: 3}
+  m_Name: 
+  m_EditorClassIdentifier: 
+  targetEnemyUnitPrefab: {fileID: 1752592239155102, guid: e1eaa60e8b4a212499dd3fe979c8d6e2,
+>>>>>>> 0add8808888a7ca2fba02d9b1f3db450008863fb
     type: 2}
   menuItemSprite: {fileID: 21300000, guid: d7374c0ee9b74a843915aee27156ba78, type: 3}
   initialPosition: {x: -550, y: -360}
   itemDimensions: {x: 160, y: 160}
+<<<<<<< HEAD
   killEnemyScript: {fileID: 114505619239075126, guid: 91ee514ff30225340afcc3729d4b0f75,
     type: 2}
 --- !u!114 &114854778193695506
+=======
+  killEnemyScript: {fileID: 114421328445299954}
+--- !u!114 &114628641955667000
+>>>>>>> 0add8808888a7ca2fba02d9b1f3db450008863fb
 MonoBehaviour:
   m_ObjectHideFlags: 1
   m_CorrespondingSourceObject: {fileID: 0}
   m_PrefabInternal: {fileID: 100100000}
+<<<<<<< HEAD
   m_GameObject: {fileID: 1666034286474376}
   m_Enabled: 1
   m_EditorHideFlags: 0
@@ -128,11 +206,36 @@ MonoBehaviour:
   attack: {fileID: 1800675918331624, guid: 7d7823586617e0d409a8ac31ba23a627, type: 2}
   targetsTag: PlayerUnit
 --- !u!212 &212274055380817034
+=======
+  m_GameObject: {fileID: 1709345393717716}
+  m_Enabled: 1
+  m_EditorHideFlags: 0
+  m_Script: {fileID: 11500000, guid: 652552b812b826248a189b39f935247b, type: 3}
+  m_Name: 
+  m_EditorClassIdentifier: 
+  animator: {fileID: 95498335459604692}
+  damageTextPrefab: {fileID: 1805029691286130, guid: 8459b94a9f1ed8f45a11bbbe6d6aa0d0,
+    type: 2}
+  damageTextPosition: {x: 0, y: 0}
+  health: 10
+  mana: 5
+  attack: 8
+  magic: 5
+  defense: 5
+  speed: 15
+  experience: 0
+  nextActTurn: 0
+--- !u!212 &212709489256783472
+>>>>>>> 0add8808888a7ca2fba02d9b1f3db450008863fb
 SpriteRenderer:
   m_ObjectHideFlags: 1
   m_CorrespondingSourceObject: {fileID: 0}
   m_PrefabInternal: {fileID: 100100000}
+<<<<<<< HEAD
   m_GameObject: {fileID: 1666034286474376}
+=======
+  m_GameObject: {fileID: 1709345393717716}
+>>>>>>> 0add8808888a7ca2fba02d9b1f3db450008863fb
   m_Enabled: 1
   m_CastShadows: 0
   m_ReceiveShadows: 0
@@ -167,7 +270,11 @@ SpriteRenderer:
   m_FlipX: 0
   m_FlipY: 0
   m_DrawMode: 0
+<<<<<<< HEAD
   m_Size: {x: 2.42, y: 1.78}
+=======
+  m_Size: {x: 1.56, y: 1.22}
+>>>>>>> 0add8808888a7ca2fba02d9b1f3db450008863fb
   m_AdaptiveModeThreshold: 0.5
   m_SpriteTileMode: 0
   m_WasSpriteAssigned: 1
diff --git a/Assets/Project/Scenes/Battle.unity b/Assets/Project/Scenes/Battle.unity
index 7d12ceb..81e623c 100644
--- a/Assets/Project/Scenes/Battle.unity
+++ b/Assets/Project/Scenes/Battle.unity
@@ -146,7 +146,11 @@ RectTransform:
   m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
   m_AnchorMin: {x: 0, y: 0.5}
   m_AnchorMax: {x: 0, y: 0.5}
+<<<<<<< HEAD
   m_AnchoredPosition: {x: 248, y: -437}
+=======
+  m_AnchoredPosition: {x: 269.8, y: -411}
+>>>>>>> 0add8808888a7ca2fba02d9b1f3db450008863fb
   m_SizeDelta: {x: 200, y: 14}
   m_Pivot: {x: 0, y: 0.5}
 --- !u!114 &2633927
@@ -229,7 +233,11 @@ RectTransform:
   m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
   m_AnchorMin: {x: 0, y: 0.5}
   m_AnchorMax: {x: 0, y: 0.5}
+<<<<<<< HEAD
   m_AnchoredPosition: {x: 248, y: -375.39996}
+=======
+  m_AnchoredPosition: {x: 269.8, y: -349.39996}
+>>>>>>> 0add8808888a7ca2fba02d9b1f3db450008863fb
   m_SizeDelta: {x: 200, y: 14}
   m_Pivot: {x: 0, y: 0.5}
 --- !u!114 &110042703
@@ -484,7 +492,11 @@ MonoBehaviour:
     m_PersistentCalls:
       m_Calls:
       - m_Target: {fileID: 596399111}
+<<<<<<< HEAD
         m_MethodName: tryRunning
+=======
+        m_MethodName: TryRunning
+>>>>>>> 0add8808888a7ca2fba02d9b1f3db450008863fb
         m_Mode: 1
         m_Arguments:
           m_ObjectArgument: {fileID: 0}
@@ -539,11 +551,19 @@ RectTransform:
   - {fileID: 2633926}
   - {fileID: 1219805731}
   m_Father: {fileID: 1331624443}
+<<<<<<< HEAD
   m_RootOrder: 2
   m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
   m_AnchorMin: {x: 0.5, y: 0.5}
   m_AnchorMax: {x: 0.5, y: 0.5}
   m_AnchoredPosition: {x: 17, y: 54}
+=======
+  m_RootOrder: 3
+  m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
+  m_AnchorMin: {x: 0.5, y: 0.5}
+  m_AnchorMax: {x: 0.5, y: 0.5}
+  m_AnchoredPosition: {x: 17, y: -23}
+>>>>>>> 0add8808888a7ca2fba02d9b1f3db450008863fb
   m_SizeDelta: {x: 100, y: 100}
   m_Pivot: {x: 0.5, y: 0.5}
 --- !u!1 &959995238
@@ -588,6 +608,39 @@ Transform:
   m_Children: []
   m_Father: {fileID: 0}
   m_RootOrder: 4
+<<<<<<< HEAD
+=======
+  m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
+--- !u!1 &1070841596
+GameObject:
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInternal: {fileID: 0}
+  serializedVersion: 6
+  m_Component:
+  - component: {fileID: 1070841597}
+  - component: {fileID: 1070841599}
+  - component: {fileID: 1070841598}
+  m_Layer: 5
+  m_Name: Overlay
+  m_TagString: Untagged
+  m_Icon: {fileID: 0}
+  m_NavMeshLayer: 0
+  m_StaticEditorFlags: 0
+  m_IsActive: 1
+--- !u!224 &1070841597
+RectTransform:
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInternal: {fileID: 0}
+  m_GameObject: {fileID: 1070841596}
+  m_LocalRotation: {x: 0, y: 0, z: 0, w: 1}
+  m_LocalPosition: {x: 0, y: 0, z: 0}
+  m_LocalScale: {x: 1, y: 1, z: 1}
+  m_Children: []
+  m_Father: {fileID: 1331624443}
+  m_RootOrder: 0
+>>>>>>> 0add8808888a7ca2fba02d9b1f3db450008863fb
   m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
 --- !u!1 &1197516865
 GameObject:
@@ -621,8 +674,13 @@ RectTransform:
   m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
   m_AnchorMin: {x: 0.5, y: 0.5}
   m_AnchorMax: {x: 0.5, y: 0.5}
+<<<<<<< HEAD
   m_AnchoredPosition: {x: 298, y: -347}
   m_SizeDelta: {x: 200, y: 42.8}
+=======
+  m_AnchoredPosition: {x: 319.8, y: -321.89996}
+  m_SizeDelta: {x: 200, y: 41}
+>>>>>>> 0add8808888a7ca2fba02d9b1f3db450008863fb
   m_Pivot: {x: 0.5, y: 0.5}
 --- !u!114 &1197516867
 MonoBehaviour:
@@ -696,8 +754,13 @@ RectTransform:
   m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
   m_AnchorMin: {x: 0.5, y: 0.5}
   m_AnchorMax: {x: 0.5, y: 0.5}
+<<<<<<< HEAD
   m_AnchoredPosition: {x: 298, y: -406.2}
   m_SizeDelta: {x: 200, y: 47.591}
+=======
+  m_AnchoredPosition: {x: 319.8, y: -383.5}
+  m_SizeDelta: {x: 200, y: 41}
+>>>>>>> 0add8808888a7ca2fba02d9b1f3db450008863fb
   m_Pivot: {x: 0.5, y: 0.5}
 --- !u!114 &1219805732
 MonoBehaviour:
@@ -842,7 +905,7 @@ RectTransform:
   m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
   m_AnchorMin: {x: 0.5, y: 0.5}
   m_AnchorMax: {x: 0.5, y: 0.5}
-  m_AnchoredPosition: {x: 526, y: -365}
+  m_AnchoredPosition: {x: 528.2, y: -360.6}
   m_SizeDelta: {x: 186, y: 188}
   m_Pivot: {x: 0.5, y: 0.5}
 --- !u!114 &1291837463
@@ -1147,7 +1210,11 @@ MonoBehaviour:
   m_OnClick:
     m_PersistentCalls:
       m_Calls:
+<<<<<<< HEAD
       - m_Target: {fileID: 0}
+=======
+      - m_Target: {fileID: 11500000, guid: 56ec9e3e0c8d94f42b5b8a3c1c676f96, type: 3}
+>>>>>>> 0add8808888a7ca2fba02d9b1f3db450008863fb
         m_MethodName: 
         m_Mode: 1
         m_Arguments:
@@ -1180,6 +1247,7 @@ GameObject:
   serializedVersion: 6
   m_Component:
   - component: {fileID: 1488873733}
+  - component: {fileID: 1488873734}
   m_Layer: 5
   m_Name: EnemyUnitsMenu
   m_TagString: EnemyUnit
@@ -1205,6 +1273,19 @@ RectTransform:
   m_AnchoredPosition: {x: -132, y: 23}
   m_SizeDelta: {x: 100, y: 100}
   m_Pivot: {x: 0.5, y: 0.5}
+--- !u!114 &1488873734
+MonoBehaviour:
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInternal: {fileID: 0}
+  m_GameObject: {fileID: 1488873732}
+  m_Enabled: 1
+  m_EditorHideFlags: 0
+  m_Script: {fileID: 11500000, guid: bc6b6267ca7bbef47998d6dd05f9fd3d, type: 3}
+  m_Name: 
+  m_EditorClassIdentifier: 
+  attackPrefab: {fileID: 0}
+  targetsTag: 
 --- !u!1 &1672219445
 GameObject:
   m_ObjectHideFlags: 0
@@ -1400,12 +1481,20 @@ MonoBehaviour:
   m_OnClick:
     m_PersistentCalls:
       m_Calls:
+<<<<<<< HEAD
       - m_Target: {fileID: 0}
+=======
+      - m_Target: {fileID: 11500000, guid: 56ec9e3e0c8d94f42b5b8a3c1c676f96, type: 3}
+>>>>>>> 0add8808888a7ca2fba02d9b1f3db450008863fb
         m_MethodName: 
         m_Mode: 1
         m_Arguments:
           m_ObjectArgument: {fileID: 0}
+<<<<<<< HEAD
           m_ObjectArgumentAssemblyTypeName: 
+=======
+          m_ObjectArgumentAssemblyTypeName: UnityEngine.Object, UnityEngine
+>>>>>>> 0add8808888a7ca2fba02d9b1f3db450008863fb
           m_IntArgument: 0
           m_FloatArgument: 0
           m_StringArgument: 
@@ -1458,6 +1547,10 @@ RectTransform:
   m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
   m_AnchorMin: {x: 0.5, y: 0.5}
   m_AnchorMax: {x: 0.5, y: 0.5}
+<<<<<<< HEAD
   m_AnchoredPosition: {x: -24, y: -347}
+=======
+  m_AnchoredPosition: {x: 66.4, y: -378}
+>>>>>>> 0add8808888a7ca2fba02d9b1f3db450008863fb
   m_SizeDelta: {x: 100, y: 100}
   m_Pivot: {x: 0.5, y: 0.5}
diff --git a/Assets/Project/Scenes/Town.unity b/Assets/Project/Scenes/Town.unity
index dcdb8f8..0e9b66b 100644
--- a/Assets/Project/Scenes/Town.unity
+++ b/Assets/Project/Scenes/Town.unity
@@ -113,17 +113,24 @@ NavMeshSettings:
     debug:
       m_Flags: 0
   m_NavMeshData: {fileID: 0}
---- !u!1 &1272416656
+--- !u!1 &234659475
 GameObject:
   m_ObjectHideFlags: 0
   m_CorrespondingSourceObject: {fileID: 0}
   m_PrefabInternal: {fileID: 0}
   serializedVersion: 6
   m_Component:
+<<<<<<< HEAD
   - component: {fileID: 1272416659}
   - component: {fileID: 1272416657}
   - component: {fileID: 1272416658}
   - component: {fileID: 1272416660}
+=======
+  - component: {fileID: 234659478}
+  - component: {fileID: 234659477}
+  - component: {fileID: 234659476}
+  - component: {fileID: 234659479}
+>>>>>>> 0add8808888a7ca2fba02d9b1f3db450008863fb
   m_Layer: 0
   m_Name: SnakeSpawner
   m_TagString: Untagged
@@ -131,12 +138,12 @@ GameObject:
   m_NavMeshLayer: 0
   m_StaticEditorFlags: 0
   m_IsActive: 1
---- !u!61 &1272416657
+--- !u!61 &234659476
 BoxCollider2D:
   m_ObjectHideFlags: 0
   m_CorrespondingSourceObject: {fileID: 0}
   m_PrefabInternal: {fileID: 0}
-  m_GameObject: {fileID: 1272416656}
+  m_GameObject: {fileID: 234659475}
   m_Enabled: 1
   m_Density: 1
   m_Material: {fileID: 0}
@@ -156,12 +163,12 @@ BoxCollider2D:
   serializedVersion: 2
   m_Size: {x: 0.25, y: 0.25}
   m_EdgeRadius: 0
---- !u!212 &1272416658
+--- !u!212 &234659477
 SpriteRenderer:
   m_ObjectHideFlags: 0
   m_CorrespondingSourceObject: {fileID: 0}
   m_PrefabInternal: {fileID: 0}
-  m_GameObject: {fileID: 1272416656}
+  m_GameObject: {fileID: 234659475}
   m_Enabled: 1
   m_CastShadows: 0
   m_ReceiveShadows: 0
@@ -202,25 +209,29 @@ SpriteRenderer:
   m_WasSpriteAssigned: 1
   m_MaskInteraction: 0
   m_SpriteSortPoint: 0
---- !u!4 &1272416659
+--- !u!4 &234659478
 Transform:
   m_ObjectHideFlags: 0
   m_CorrespondingSourceObject: {fileID: 0}
   m_PrefabInternal: {fileID: 0}
-  m_GameObject: {fileID: 1272416656}
+  m_GameObject: {fileID: 234659475}
   m_LocalRotation: {x: 0, y: 0, z: 0, w: 1}
+<<<<<<< HEAD
   m_LocalPosition: {x: -3.86, y: -2.95, z: 0}
+=======
+  m_LocalPosition: {x: -1.91, y: -1.25, z: 59.16358}
+>>>>>>> 0add8808888a7ca2fba02d9b1f3db450008863fb
   m_LocalScale: {x: 2, y: 2, z: 2}
   m_Children: []
   m_Father: {fileID: 0}
   m_RootOrder: 3
   m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
---- !u!114 &1272416660
+--- !u!114 &234659479
 MonoBehaviour:
   m_ObjectHideFlags: 0
   m_CorrespondingSourceObject: {fileID: 0}
   m_PrefabInternal: {fileID: 0}
-  m_GameObject: {fileID: 1272416656}
+  m_GameObject: {fileID: 234659475}
   m_Enabled: 1
   m_EditorHideFlags: 0
   m_Script: {fileID: 11500000, guid: a80bba4c6acc8724f89ad585eaf72a0f, type: 3}
diff --git a/Assets/Project/Sprites/battle/bat.png.meta b/Assets/Project/Sprites/battle/bat.png.meta
index d8452d5..c841b54 100644
--- a/Assets/Project/Sprites/battle/bat.png.meta
+++ b/Assets/Project/Sprites/battle/bat.png.meta
@@ -106,7 +106,11 @@ TextureImporter:
       physicsShape: []
       tessellationDetail: 0
       bones: []
+<<<<<<< HEAD
       spriteID: 9c9187214ae99cc4f9c9b2b2bcfe2178
+=======
+      spriteID: a503049dba6d6ec499726c48225f3f91
+>>>>>>> 0add8808888a7ca2fba02d9b1f3db450008863fb
       vertices: []
       indices: 
       edges: []
@@ -126,7 +130,11 @@ TextureImporter:
       physicsShape: []
       tessellationDetail: 0
       bones: []
+<<<<<<< HEAD
       spriteID: eb9e991148ab27c41bdaa47c758c4680
+=======
+      spriteID: c0f2a3877482fc8448e755ef32a6f74a
+>>>>>>> 0add8808888a7ca2fba02d9b1f3db450008863fb
       vertices: []
       indices: 
       edges: []
@@ -146,7 +154,11 @@ TextureImporter:
       physicsShape: []
       tessellationDetail: 0
       bones: []
+<<<<<<< HEAD
       spriteID: 0edc88aeacab11e46ad6be9bf5cf4d51
+=======
+      spriteID: 38f7dfdf81a2ab0489d806aafab8fbd0
+>>>>>>> 0add8808888a7ca2fba02d9b1f3db450008863fb
       vertices: []
       indices: 
       edges: []
@@ -166,7 +178,11 @@ TextureImporter:
       physicsShape: []
       tessellationDetail: 0
       bones: []
+<<<<<<< HEAD
       spriteID: f32c4d49d2af8aa4cad6c091e6aa54a9
+=======
+      spriteID: 58bfb0196a08caf4e8499e62566dcca8
+>>>>>>> 0add8808888a7ca2fba02d9b1f3db450008863fb
       vertices: []
       indices: 
       edges: []
@@ -186,7 +202,11 @@ TextureImporter:
       physicsShape: []
       tessellationDetail: 0
       bones: []
+<<<<<<< HEAD
       spriteID: b8304a075649d6541bb206b182f7fec7
+=======
+      spriteID: 34368beccc7476641865b19305e2e45a
+>>>>>>> 0add8808888a7ca2fba02d9b1f3db450008863fb
       vertices: []
       indices: 
       edges: []
@@ -206,7 +226,11 @@ TextureImporter:
       physicsShape: []
       tessellationDetail: 0
       bones: []
+<<<<<<< HEAD
       spriteID: c4288c362a99b9e41b22af533f5b3781
+=======
+      spriteID: 2172563071902ed4398b3f03cac205f1
+>>>>>>> 0add8808888a7ca2fba02d9b1f3db450008863fb
       vertices: []
       indices: 
       edges: []
@@ -226,7 +250,11 @@ TextureImporter:
       physicsShape: []
       tessellationDetail: 0
       bones: []
+<<<<<<< HEAD
       spriteID: 55b37503f8fc723479ef2bf59d7a8cf8
+=======
+      spriteID: 6582336757fd58a4890dae166baf0978
+>>>>>>> 0add8808888a7ca2fba02d9b1f3db450008863fb
       vertices: []
       indices: 
       edges: []
@@ -246,7 +274,11 @@ TextureImporter:
       physicsShape: []
       tessellationDetail: 0
       bones: []
+<<<<<<< HEAD
       spriteID: 0b2fde1338a76c84cb3be9ce26244eee
+=======
+      spriteID: 713249bbefa241d46b1a2bd11370b443
+>>>>>>> 0add8808888a7ca2fba02d9b1f3db450008863fb
       vertices: []
       indices: 
       edges: []
@@ -266,7 +298,11 @@ TextureImporter:
       physicsShape: []
       tessellationDetail: 0
       bones: []
+<<<<<<< HEAD
       spriteID: c453f5917eefc8e47b1e152523bb2be8
+=======
+      spriteID: 4b76e179c6912e44ab0cd68bb55085bd
+>>>>>>> 0add8808888a7ca2fba02d9b1f3db450008863fb
       vertices: []
       indices: 
       edges: []
@@ -286,7 +322,11 @@ TextureImporter:
       physicsShape: []
       tessellationDetail: 0
       bones: []
+<<<<<<< HEAD
       spriteID: b268b6564bec0cd46a8ac7085737ab80
+=======
+      spriteID: fb3b1b81f040b084793af432395225ee
+>>>>>>> 0add8808888a7ca2fba02d9b1f3db450008863fb
       vertices: []
       indices: 
       edges: []
@@ -306,7 +346,11 @@ TextureImporter:
       physicsShape: []
       tessellationDetail: 0
       bones: []
+<<<<<<< HEAD
       spriteID: df82a7f46300f944bb97e1004fb99b6b
+=======
+      spriteID: a39b824a80eb9a14ba9d89a5c18c7c15
+>>>>>>> 0add8808888a7ca2fba02d9b1f3db450008863fb
       vertices: []
       indices: 
       edges: []
@@ -326,7 +370,11 @@ TextureImporter:
       physicsShape: []
       tessellationDetail: 0
       bones: []
+<<<<<<< HEAD
       spriteID: e2e6e9f76ad434a458a0e4c4fa0c5251
+=======
+      spriteID: ff6a6f1b3562d6845bcd774324c19792
+>>>>>>> 0add8808888a7ca2fba02d9b1f3db450008863fb
       vertices: []
       indices: 
       edges: []
@@ -346,7 +394,11 @@ TextureImporter:
       physicsShape: []
       tessellationDetail: 0
       bones: []
+<<<<<<< HEAD
       spriteID: fc12db512ec5554438e933daa7121caf
+=======
+      spriteID: f7a64110f26b5ee468ade4c2fe401f0c
+>>>>>>> 0add8808888a7ca2fba02d9b1f3db450008863fb
       vertices: []
       indices: 
       edges: []
@@ -366,7 +418,11 @@ TextureImporter:
       physicsShape: []
       tessellationDetail: 0
       bones: []
+<<<<<<< HEAD
       spriteID: 90948e5558f12ba4cb5be30cf33e3f89
+=======
+      spriteID: 098c7d3fc2eda834c8f4a489c229861a
+>>>>>>> 0add8808888a7ca2fba02d9b1f3db450008863fb
       vertices: []
       indices: 
       edges: []
@@ -386,7 +442,11 @@ TextureImporter:
       physicsShape: []
       tessellationDetail: 0
       bones: []
+<<<<<<< HEAD
       spriteID: b566f8ca43e2b1d409172cef3763eaba
+=======
+      spriteID: 29165e85220afec4f9ae5a0d9f3b33c7
+>>>>>>> 0add8808888a7ca2fba02d9b1f3db450008863fb
       vertices: []
       indices: 
       edges: []
@@ -406,7 +466,11 @@ TextureImporter:
       physicsShape: []
       tessellationDetail: 0
       bones: []
+<<<<<<< HEAD
       spriteID: 166ad696227459f48a86cea7f1ae81cc
+=======
+      spriteID: 08c3a72b00cf0d3409e3863955252485
+>>>>>>> 0add8808888a7ca2fba02d9b1f3db450008863fb
       vertices: []
       indices: 
       edges: []
@@ -426,7 +490,11 @@ TextureImporter:
       physicsShape: []
       tessellationDetail: 0
       bones: []
+<<<<<<< HEAD
       spriteID: bbf9959b5b5b4e14c937edd3940aa52c
+=======
+      spriteID: 26ddc1523a38d3444a43803ace5f4d48
+>>>>>>> 0add8808888a7ca2fba02d9b1f3db450008863fb
       vertices: []
       indices: 
       edges: []
@@ -446,7 +514,11 @@ TextureImporter:
       physicsShape: []
       tessellationDetail: 0
       bones: []
+<<<<<<< HEAD
       spriteID: 73443ce8a63d04947a68d370619ae458
+=======
+      spriteID: f1c91c07f8f2a1448851d21e7906726b
+>>>>>>> 0add8808888a7ca2fba02d9b1f3db450008863fb
       vertices: []
       indices: 
       edges: []
@@ -466,7 +538,11 @@ TextureImporter:
       physicsShape: []
       tessellationDetail: 0
       bones: []
+<<<<<<< HEAD
       spriteID: 3a4a50f501140a349b391cafb277a216
+=======
+      spriteID: de7ee2633e2864f498af5f248ea9ca68
+>>>>>>> 0add8808888a7ca2fba02d9b1f3db450008863fb
       vertices: []
       indices: 
       edges: []
@@ -474,7 +550,11 @@ TextureImporter:
     outline: []
     physicsShape: []
     bones: []
+<<<<<<< HEAD
     spriteID: b1f0efcb270b84c47b04fbd0ef5dded7
+=======
+    spriteID: 5359f50a7826cd64898211138c725438
+>>>>>>> 0add8808888a7ca2fba02d9b1f3db450008863fb
     vertices: []
     indices: 
     edges: []
